README.md

```diff
@@ -1 +1,6 @@
 # ConnectedCircularity
+
+### Docker Compose 
```

Comment:
Maybe add note that you should add .env file for each service before.

Reply:
Done